tests/conftest.py

```diff
@@ -52,26 +58,51 @@ def create_client(self):
         command = ['xterm', '-hold', '-class', wmclass, '-e', 'true']
         # enforce a hook when the window appears
         self.call('rule', 'once', 'class='+wmclass, 'hook=here_is_'+wmclass)
-        # run process that wait for the hook
+        # the following is still racy because xterm might connect to X before
```

Comment:
Could we start a single, persistent `hc --idle` along with hlwm to avoid this race condition?